src/Apps/Order/Components/ItemReview.tsx

```diff
@@ -34,7 +34,7 @@ export const ItemReview: React.SFC<ItemReviewProps> = ({
     },
   },
 }) => (
-  <StackableBorderBox>
+  <BorderBox>
```

Comment:
This is #minor but I am pretty sure every place we show the `ItemReview` it will be encased in a border, so we don't need the `StackableBorderBox`.

Reply:
Makes sense :+1: but BorderBox doesn't have responsive padding by default, so let's add it back with `p={[2, 3]}`

Reply:
Oooh! Didn't realize that, thanks! 🙌